packages/gatsby-core-utils/src/ci.ts

```diff
@@ -4,6 +4,8 @@ const CI_DEFINITIONS = [
   envFromCIandCIName,
   getEnvDetect({ key: `NOW_BUILDER_ANNOTATE`, name: `ZEIT Now` }),
   getEnvDetect({ key: `NOW_REGION`, name: `ZEIT Now v1` }),
+  getEnvDetect({ key: `VERCEL_URL`, name: `Vercel Now` }),
+  getEnvDetect({ key: `NOW_BUILDER`, name: `Vercel Now` }),
```

Comment:
when i look at this doc: the `NOW_BUILDER` is also renamed to `VERCEL_BUILDER`:

https://www.notion.so/Change-Timeline-fab7622780694753a92c0a396cea7eb6

Reply:
cc #24019

Reply:
they still expose NOW_BUILDER 🙈

Reply:
tho why not add both should they one day fix it

Reply:
https://github.com/gatsbyjs/gatsby/pull/25926

Reply:
> Existing projects will get both NOW_ and VERCEL_ env. New projects will only get VERCEL_ env.

there is no info about an expire

Reply:
> they still expose NOW_BUILDER 🙈

only on existing apps

Reply:
I'd create a new app fwiw

Reply:
👍